Documentation/deploy-workers.md

```diff
@@ -86,11 +72,21 @@ Create `/etc/systemd/system/kubelet.service` and substitute the following variab
 
 ```yaml
 [Service]
-ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
-
 Environment=KUBELET_VERSION=${K8S_VER}
+Environment="RKT_OPTS=--volume cni,kind=host,source=/opt/cni/bin --mount volume=cni,target=/opt/cni/bin"
+ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
+ExecStartPre=/usr/bin/mkdir -p /opt/cni/bin
+ExecStartPre=/usr/bin/chmod a+w /opt/cni/bin
+ExecStartPre=-/usr/bin/wget -nc -O /opt/cni/bin/calico https://github.com/projectcalico/calico-cni/releases/download/v1.3.0/calico
+ExecStartPre=-/usr/bin/wget -nc -O /opt/cni/bin/flannel https://f001.backblaze.com/file/calico/flannel
+ExecStartPre=-/usr/bin/wget -nc -O /opt/cni/bin/host-local https://f001.backblaze.com/file/calico/host-local
+ExecStartPre=/usr/bin/chmod +x /opt/cni/bin/calico
+ExecStartPre=/usr/bin/chmod +x /opt/cni/bin/flannel
+ExecStartPre=/usr/bin/chmod +x /opt/cni/bin/host-local
```

Comment:
You mention this in the PR, but wget'ing a bunch of binaries to the host isn't an ideal method for distributing these assets. Might need to think about this a bit more, but options might be including blessed versions/assets in the hyperkube image. Or at least distributing these binaries via utility containers. Or? 

/cc @philips